sql/data_node.sql

```diff
@@ -10,3 +10,6 @@ CREATE OR REPLACE FUNCTION _timescaledb_internal.remote_txn_heal_data_node(forei
 RETURNS INT
 AS '@MODULE_PATHNAME@', 'ts_remote_txn_heal_data_node'
 LANGUAGE C STRICT;
+
+CREATE OR REPLACE PROCEDURE _timescaledb_internal.data_node_exec(node_name NAME, command TEXT)
```

Comment:
Why are we exposing it, for testing?

Reply:
yes, for testing. I can limit it to debug builds I guess.